### PR TITLE
Feature/email analytics results

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/AnalyticsBundle/Service/AnalyticsCSVCreationService.php [hunk 1]
```diff
@@ -35,20 +35,28 @@ public function getSerializer()
...
     /**
      * @param AnalyticsExport $export
+     * @return array
      */
     public function createAndPopulateCSV(AnalyticsExport $export)
     {
-        if (false !== ($handle = $this->openFile($this->generateFileNameAndPath($export), 'w+'))) {
+        $filePath = $this->generateFileNameAndPath($export);
+
+        if (false !== ($handle = $this->openFile($filePath, 'w+'))) {
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Service/AnalyticsCSVCreationService.php]: Does not read nicely

File: src/TransformCore/PHE/AnalyticsBundle/Service/EmailService.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    {
+        $this->mailer = null;
+    }
+
+    /**
+     * @param string $subject
+     * @param string $body
+     * @param array  $attachments
+     *
+     * @return array<string, int>
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Service/EmailService.php]: :+1:

File: src/TransformCore/PHE/AnalyticsBundle/Service/EmailService.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            $this->mailer = \Swift_Mailer::newInstance($this->transport);
+        }
+
+        return $this->mailer;
+    }
+
+    /**
+     * EmailService constructor.
+     */
+    public function __construct()
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Service/EmailService.php]: This should probably be the first method not to get lost

File: src/TransformCore/PHE/AnalyticsBundle/Service/AnalyticsCSVCreationService.php (hunk 1 above)
Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Service/AnalyticsCSVCreationService.php]: Reads better than it did :)

**Pull request merged**